### PR TITLE
Add Google Tag Manager and cookie consent

### PR DESCRIPTION
**Pull request by amakarudze**
Changes in this PR:
- Add a tag for Google Analytics to enable tracking website visits.
- Add cookie consent for users.

The cookie consent dialogue looks as below:
Screenshot 2024-02-05 at 17 13 32

### CODE REVIEW

File: templates/event/base.html [hunk 1]
```diff
@@ -3,6 +3,14 @@
 <!DOCTYPE HTML>
 <html>
 <head>
+  <!-- Google tag (gtag.js) -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-JGJW1VEY0D"></script>
```
Comment (marksweb) [inline templates/event/base.html:7]: This gets loaded regardless of the cookie consent choice, so what functionality is listening out for that consent?

Comment (amakarudze) [reply templates/event/base.html:7]: Thanks, let me add that functionality.

Comment (amakarudze) [reply templates/event/base.html:7]: @marksweb, I followed the guidelines here https://developers.google.com/analytics/devguides/collection/gtagjs/user-opt-out to disable the google tag on cookie reject.

**Review by marksweb — APPROVED** [review]
Ok, that looks good 👍

**Pull request merged**